mmv1/templates/terraform/examples/firestore_database.tf.erb

```diff
@@ -0,0 +1,37 @@
+resource "google_project" "project" {
+	provider = "google-beta"
+
+	project_id = "tf-test%{random_suffix}"
+	name       = "tf-test%{random_suffix}"
+	org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "time_sleep" "wait_60_seconds" {
+  depends_on = [google_project.project]
+
+  create_duration = "60s"
+}
+
+resource "google_project_service" "firestore" {
+  provider = "google-beta"
+
+  project = google_project.project.project_id
+  service = "firestore.googleapis.com"
+
+  # Needed for CI tests for permissions to propagate, should not be needed for actual usage
+  depends_on = [time_sleep.wait_60_seconds]
+}
+
+resource "google_firestore_database" "<%= ctx[:primary_resource_id] %>" {
+    provider = "google-beta"
+
+    project                     = google_project.project.project_id
+    name                        = "(default)"
```

Comment:
We may want to use a variable here and pass it in from terraform.yaml, so that the example can be `(default)`, but the test can be something that we can identify and clean up, like `tf-test-database`. Otherwise, we might run into issues with tests where this database already exists (if this is a singleton resources, then perhaps this doesn't apply).

Reply:
As of this writing, "(default)" is the only supported database ID.

Reply:
Also worth noting that database deletion is not currently supported, which is why all the tests spin up their own Cloud project, enable the Firestore API, create/update the database, and then delete the whole thing.

Reply:
That works, thanks for explaining!